modules/nw-multus-vlan-object.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+//37.1. VLAN overview
+//
+:_content-type: REFERENCE
+[id="nw-multus-vlan-object_{context}"]
+= Configuration for an VLAN additional network
+
+The following object describes the configuration parameters for the VLAN CNI plugin:
+
+.VLAN CNI plugin JSON configuration object
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+|`cniVersion`
+|`string`
+|The CNI specification version. The `0.3.1` value is required.
+
+|`name`
+|`string`
+|The value for the `name` parameter you provided previously for the CNO configuration.
+
+|`type`
+|`string`
+|The name of the CNI plugin to configure: `vlan`.
+
+|`master`
+|`string`
+|The Ethernet interface to associate with the network attachment. If a `master` is not specified, the interface for the default network route is used.
+
+|`mtu`
+|`integer`
+|Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
+
+|`vlanId`
+|`integer`
+|Set the id of the vlan.
```

Comment:
above mtu? it would be nice to have required parameters first and then the optional ones